documentation/docs/01-routing.md

```diff
@@ -51,37 +51,26 @@ Endpoints are modules written in `.js` (or `.ts`) files that export functions co
 // Declaration types for Endpoints
 // * declarations that are not exported are for internal use
 
-// type of string[] is only for set-cookie
-// everything else must be a type of string
-type ResponseHeaders = Record<string, string | string[]>;
-
 export interface RequestEvent<Locals = Record<string, any>> {
 	request: Request;
 	url: URL;
 	params: Record<string, string>;
 	locals: Locals;
 }
 
-type Body = JSONResponse | Uint8Array | string | ReadableStream | stream.Readable;
-
+type Body = JSONString | Uint8Array | ReadableStream | stream.Readable;
```

Comment:
why was `string` removed?

Reply:
It's already included as part of `JSON` basic types, from `JSONString`